content/getting-started/icerpc-protobuf-tutorial/client-tutorial.md

```diff
@@ -63,11 +63,35 @@ The main program starts by creating a connection to the server:
 using ILoggerFactory loggerFactory = LoggerFactory.Create(builder =>
     builder
         .AddSimpleConsole()
-        .AddFilter("IceRpc", LogLevel.Debug));
+        .AddFilter("IceRpc", LogLevel.Information));
```

Comment:
The documentation doesn't match the actual templates. This was changed in https://github.com/icerpc/icerpc-csharp/issues/3840